.github/workflows/security.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: security
+
+on:
+  push:
+    branches: [development/*, stabilization/*, hotfix/*]
+  release:
+    types: [published]
+  pull_request:
+    branches: [development/*, stabilization/*, hotfix/*]
+  schedule:
+    - cron: '0 8 * * 1' # Monday - 8am - UTC
+  workflow_dispatch:
+
+
+jobs:
+  build-zenko-ui:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    with:
+      name: zenko-ui
+  
+  build-keycloak:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    with:
+      name: ci-keycloak
+
+  trivy-zenko-ui:
+    needs: build-zenko-ui
+    name: Trivy Vulnerability Scan for Code for Zenko-ui
+    uses: scality/workflows/.github/workflows/trivy.yaml@v1
+    with:
+      name: zenko-ui
+
+  trivy-keycloak:
+    needs: build-keycloak
+    name: Trivy Vulnerability Scan for Code
+    uses: scality/workflows/.github/workflows/trivy.yaml@v1
+    with:
+      name: ci-keycloak
```

Comment:
I don't think we would want to scan this image. it's only used for running automatic tests and not bundled in any release.

Reply:
Agree